asapdiscovery-data/asapdiscovery/data/schema/ligand.py

```diff
@@ -617,6 +714,34 @@ def get_chemical_relationship(self, other: "Ligand") -> ChemicalRelationship:
             relationship |= ChemicalRelationship.DISTINCT
         return relationship
 
+    def sort_confs(self, by: str, ascending: bool = True) -> np.ndarray:
```

Comment:
This is great, maybe make it super clear that in the doc string the ligand is edited in place.